packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -67,6 +71,19 @@ export default function Sidebar({
   const showSettings = modal?.type === LazyModal.UserSettings;
   const { isNewMobileLayout } = useMobileUxExperiment();
   const isTablet = useViewSize(ViewSize.Tablet);
+  const sidebarToggle = useFeature(feature.sidebarToggle);
+  const { completeAction, checkHasCompleted } = useActions();
```

Comment:
Why exactly do we have an action for this?

Reply:
Because we don't want to persist the toggle being closed (if they end up on that value for the experiment) if they decide to manually toggle it open.

Reply:
I think @capJavert idea above solves this as well, let's see if we can consider going on that direction